lib/std/os/linux.zig

```diff
@@ -7290,7 +7349,7 @@ pub const AUDIT = struct {
         ARMEB = toAudit(.armeb),
         CSKY = toAudit(.csky),
         HEXAGON = @intFromEnum(std.elf.EM.HEXAGON),
-        X86 = toAudit(.x86),
+        LOONGARCH64 = toAudit(.loongarch64),
```

Comment:
Just noting that I plan to rewrite these audit bindings in a follow-up PR. There's no sense in doing `std.Target.Cpu.Arch` -> `std.elf.EM` here, especially since there is not a 1:1 mapping.